apps/vmq_gojek_auth/src/vmq_gojek_auth.erl

```diff
@@ -286,9 +312,38 @@ in(Type, User, Topic) when is_binary(Topic) ->
             error_logger:warning_msg("can't validate ~p acl topic ~p for user ~p due to ~p", [Type, STopic, User, Reason])
     end.
 
+insert_token(Type, _, Topic) when is_binary(Topic) ->
+  Words = parse_topic(Topic),
```

Comment:
Parse topic can return ok incase of topic validation error. Can we handle it to not insert it into the tbl?

Reply:
Returning `[]` from parse_topic now